test/system/252-quadlet.bats

```diff
@@ -153,6 +153,11 @@ Notify=yes
 LogDriver=passthrough
 EOF
 
+    # FIXME: Temporary until podman fully removes cgroupsv1 support; see #21431
+    if [[ -n "$IGNORE_CGROUPSV1_WARNING" ]]; then
+        echo "Environment=IGNORE_CGROUPSV1_WARNING=1" >>$quadlet_file
+    fi
```

Comment:
Well, so much for that idea. I guess quadlet sends this to the container itself, and the podman command never sees it. Back to the drawing board.

Reply:
you would need to set the environment section under the systemd [Service] section not [Container] as this maps to --env which of course does not effect podman itself.